app/views/main/index.html.erb

```diff
@@ -0,0 +1 @@
+<h1>calling from main</h1>
```

Comment:
- [ ] Kindly remove this main views file as we have removed the `main` route and controller for it.

Reply:
well resolved! ✔️